python/python/bin/ert_non_interactive

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+import sys
+
+from res.enkf import EnKFMain, ResConfig, ESUpdate, ErtRunContext
+from ecl.util.util import BoolVector
+
+import argparse
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument("--log-file", default="ert-server.log", dest="log_file")
+    parser.add_argument("--log-level", type=int, default=1, dest="log_level")
+    parser.add_argument("config_file")
+
+    args = parser.parse_args()
+    log_level = args.log_level
+
+    config_file = args.config_file
+    config = ResConfig(config_file)
+
+    ert = EnKFMain(config)
+    fs_manager = ert.getEnkfFsManager()
+    es_update = ESUpdate(ert)
+
+    result_fs = fs_manager.getCurrentFileSystem()
+    target_fs = fs_manager.getFileSystem("default")
+
+    model_config = ert.getModelConfig()
+    runpath_fmt = model_config.getRunpathFormat()
+    jobname_fmt = model_config.getJobnameFormat()
+    subst_list = ert.getDataKW()
+    itr = 0
+
+    mask = BoolVector(default_value=False)
```

Comment:
This is purely a WIP and should not be merged yet. I only pushed out a PR early on.. 

So far I've pretty much looked through how it could be done, influenced by how it's done other places in the code.. However it's out of context of course, so it's likely incorrect.

I do appreciate the comments though. The reason it's done like that was pretty much because I wasn't sure where to get the mask from, so just to get it to run. I'll look at what you suggested.

I believe it'll be more beneficial with a discussions once I've done a bit more and have better questions to ask.